src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* Experimental (only in pre-release versions): Added the `EventName` property
```

Comment:
nit: This should be added below the existing entries in the "Unreleased" section